### DISCUSSION TITLE
Joint tests / test equallity of treatment coefficients

### DISCUSSION THREAD

**Discussion by paramobrotz** [DoubleML/doubleml-for-r#189] [Q&A] [answered]
I have two related questions:
1)  When estimating coefficients for more than one treatment variable (for example with DoubleMLPLR), is it possible to get p-values for joint tests such as T1 = T2 using the built-in [bootstrap procedure](https://docs.doubleml.org/stable/guide/se_confint.html), or an alternative method?
2) Is there a way to conduct similar tests for clustered data? 

Thank you!

**Comment by PhilippBach** [comment, marked as answer]
Hi @paramobrotz ,

my apologies for this late reply. Regarding 1. I think the p-values for our coefficient tests typically refer to the null hypothesis $H_0: \theta_{0}=0$. Basically it should be possible to adjust this (e.g., by setting the same value for the null hypothesis for T1 and T2). Probably, we'd have to add an option to provide values for the null, when calling `bootstrap()` and `p_adjust()`... It's not implemented yet, so we'd have to add it as a new feature.

Regarding 2. We currently only have implemented the analytical standard errors (etc) for clustered data, but I see that this is relevant (see https://github.com/DoubleML/doubleml-for-r/discussions/190) . I think there are some details which are not yet 100% clear on the interaction between clustered data and multiplier bootstrap and we'd have to look for a paper on that issue.

**Reply by paramobrotz** [reply]
Thank you for your answer!
